zkevm-circuits/src/table.rs

```diff
@@ -5,7 +5,7 @@ use crate::{
     evm_circuit::util::rlc,
     exp_circuit::param::{OFFSET_INCREMENT, ROWS_PER_STEP},
     impl_expr,
-    util::{build_tx_log_address, word, Challenges},
+    util::{build_tx_log_address, Challenges},
```

Comment:
The changes in this file revert all the words lo/hi in tables.  Each table in general is used in at least 2 circuits, so for circuits word lo/hi refactors, if people stick to a single circuit there will be compilation failures (because one circuit expects a table with words lo/hi and the other doesn't).  How should we proceed with this?

I think one option is to duplicate all the table types (and associated methods) and for each table have `TableLegacy` and `Table`.  Or `Table` and `TableNew`.  One with word_rlc the other with word lo/hi.  Then the refactors can be done incrementally.